qiskit/circuit/library/standard_gates/u.py

```diff
@@ -118,13 +114,12 @@ def control(
     def __array__(self, dtype=complex):
         """Return a numpy.array for the U gate."""
         theta, phi, lam = (float(param) for param in self.params)
+        cos = numpy.cos(theta / 2)
+        sin = numpy.sin(theta / 2)
```

Comment:
Per #8218 which just merged we should use the math lib for these

```suggestion
        cos = math.cos(theta / 2)
        sin = math.sin(theta / 2)
```